### ISSUE TITLE
[Bug]:  Crash on Cache loading

### ISSUE THREAD

**Issue by Savolaine** [Yimura/YimMenu#3575]
### Describe the bug

After injecting with xenos/extreme injector etc. crashes upon pressing update cache

### Steps To Reproduce

1. Inject Menu
2. Load Singeplayer
3. press update cache because it needs it
4. crash

### Expected Behavior

Start updating Cache

### Logs

[cout.log](https://github.com/user-attachments/files/16596394/cout.log)

### Store

Steam

### Language

English

### YimMenu Language

English (US)

### Additional context

I have tried deleting yimmenu's folder from appdata/roaming to no avail and the folder is weridly read only everytime I go and  check the log and changing it to not be read only results in it still being read only when going to check on it again

### Build

- [X] I've confirmed the issue exists on the latest version of YimMenu

### Coload

- [X] I've tested this without co-loading anything.

**Comment by CypherMorgan** [comment]
Could be related to #1859

**Comment by xiaoxiao921** [comment]
recursive_directory_iterator::operator++: unknown error

Lol wtf

**Comment by Savolaine** [comment]
Yeah I got no clue on what "recursive_directory_iterator::operator++: unknown error" means. have tried with av off gta stored in win boot drive and nothing has fixed it, also tested other menus and I am unable to inject anything

So the fix is basically wait and it will someday work?

**Comment by CypherMorgan** [comment]
Try this, might help.. Go to your E:\SteamLibrary\steamapps\common right click on Grand Theft Auto V folder, Properties then Edit and Allow Full Control for all users and Apply and retry, And Make sure you don't have any file in your game's directory which might conflict with ur game's files integrity. Try the same on Yimmenu's folder if it's the one in Read-Only Mode, and after that you can probably turn off read only mode as well.
